src/core/deltafilewrapper.cpp

```diff
@@ -650,13 +650,20 @@ void DeltaFileWrapper::addPatch( const QString &localLayerId, const QString &sou
   QMap<QString, int> layerPkDeltaIdx = mLocalPkDeltaIdx.value( localLayerId );
   QString localPk = delta.value( QStringLiteral( "localPk" ) ).toString();
 
+  qInfo() << "DeltaFileWrapper::addPatch: localPk=" << localPk << " layerPkDeltaIdx=" << layerPkDeltaIdx;
+
   if ( layerPkDeltaIdx.contains( localPk ) )
   {
     int deltaIdx = layerPkDeltaIdx.take( localPk );
     QJsonObject deltaCreate = mDeltas.at( deltaIdx ).toObject();
+
+    Q_ASSERT( deltaCreate.value( QStringLiteral( "method" ) ).toString() == QStringLiteral( "create" ) );
```

Comment:
This will only trigger on debug builds.
I would add the following

```
sentry_value_t exc = sentry_value_new_exception("Exception", "Error message.");
sentry_value_set_stacktrace(exc, NULL, 0);
sentry_event_add_exception(event, exc);

sentry_capture_event(event);
```

Reply:
yes, I know, it was ommited in the first place when the code was first submitted. We will get the data on sentry anyways a few lines below.